src/Root.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  HashRouter,
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/pages/HomePage';
+import { PeoplePage } from './components/pages/PeoplePage';
+import { NotFoundPage } from './components/pages/NotFoundPage';
+
+export const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="/" element={<HomePage />} />
+
+        <Route path="/home" element={<Navigate to="/" replace />} />
+
+        <Route path="/people" element={<PeoplePage />}>
+          <Route
+            path="/people/:slug?"
+            element={<PeoplePage />}
+          />
```

Comment:
As this Route is a nested route of `/people` you don't have to specify `/people/:slug?`, just `:slug?`